### PR TITLE
fix: add function web pattern and update azure SQL server name

### PR DESCRIPTION
**Pull request by wenytang-ms**
E2E TEST: https://github.com/OfficeDev/TeamsFx/actions/runs/1498494500

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12674299

### CODE REVIEW

**Review by blackchoey — COMMENTED** [review]
the bug also mentions an error about SQL provision, do we also need to fix it?

**Comment by wenytang-ms** [comment]
> the bug also mentions an error about SQL provision, do we also need to fix it?

update the azure SQL server name to fix the second provision error

File: packages/fx-core/src/core/middleware/projectMigrator.ts [hunk 1]
```diff
@@ -131,7 +131,7 @@ class EnvConfigName {
 export class ArmParameters {
   static readonly FEStorageName = "frontendHostingStorageName";
   static readonly IdentityName = "userAssignedIdentityName";
-  static readonly SQLServer = "azureSqlServerName";
+  static readonly SQLServer = "sqlServerName";
   static readonly SQLDatabase = "azureSqlDatabaseName";
```
Comment (xzf0587) [inline packages/fx-core/src/core/middleware/projectMigrator.ts]: azureSqlDatabaseName should be sqlDatabaseName

Comment (wenytang-ms) [reply packages/fx-core/src/core/middleware/projectMigrator.ts]: already updated.

**Review by xzf0587 — APPROVED** [review]

**Review by blackchoey — APPROVED** [review]

**Pull request merged**